### PR TITLE
Add url detection for articles

### PR DESCRIPTION
**Pull request by justincorrigible**

### CODE REVIEW

File: client/main.js [hunk 1]
```diff
@@ -379,6 +379,8 @@ $(document).ready(function() {
   $('#nav-wiki-btn').on('click', toggleWiki);
 
   $('.wiki-aside-action-collapse').on('click', collapseWiki);
+  $('#wikiFrame').attr('src', 'http://freecodecamp.github.io' + window.location.pathname);
```
Comment (SaintPeter) [inline client/main.js]: If we do not have an `en` in the path will this work?

Comment (hallaathrad) [inline client/main.js]: it should go to whatever is typed in the url bar. if it doesn't, it'll default to root, I believe

**Comment by BerkeleyTrue** [comment]
Hey @hallaathrad

This doesn't seem to work. The actual wiki iframe isn't created until it first opened. So the jQuery attribute method will no-op where you've placed it

**Comment by hallaathrad** [comment]
yeah, this was created based on the old implementation of the thing. We've changed a few things ever since. Let me bring this PR up to speed asap.

**Comment by BerkeleyTrue** [comment]
Closing as abandoned.

@hallaathrad Please feel free to re-open when ready.

**Pull request closed without merging**